src/NosCore.Controllers/Shops/NrunPacketHandler.cs

```diff
@@ -73,7 +73,7 @@ public override Task ExecuteAsync(NrunPacket nRunPacket, ClientSession clientSes
             }
 
             _nRunProvider.NRunLaunch(clientSession, new Tuple<IAliveEntity, NrunPacket>(aliveEntity!, nRunPacket));
-            return Task.CompletedTask;
+            return Task.Delay(10); //todo find a way to await on the observers without a async void
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5326532)